app/assets/javascripts/games.js.coffee

```diff
@@ -28,3 +28,18 @@ $ ->
 
   window.private_channel.bind 'show_forfeit_button', (event) ->
     $('.forfeit').css('display', 'inline')
+
+  window.private_channel.bind 'pawn-promotion', (event) ->
+    window.pawnPromotionUpdateURL = event.pawnPromotionUpdateURL
+    $('#message').css('display', 'block')
+    $('#message').html(event.message)
+    $('#Queen-promotion').css('display', 'inline')
+    $('#Rook-promotion').css('display', 'inline')
+    $('#Knight-promotion').css('display', 'inline')
+    $('#Bishop-promotion').css('display', 'inline')
+
+  window.private_channel.bind 'pawn-promotion-choosen', (event) ->
+    $('#Queen-promotion').css('display', 'none')
+    $('#Rook-promotion').css('display', 'none')
+    $('#Knight-promotion').css('display', 'none')
+    $('#Bishop-promotion').css('display', 'none')
```

Comment:
yeah.. it wasn't being triggered in the pawn-promotion method... just added it..